include/deal.II/base/data_out_base.h

```diff
@@ -2327,7 +2327,7 @@ namespace DataOutBase
                       const bool                               write_mesh_file,
                       const std::string &                      mesh_filename,
                       const std::string &solution_filename,
-                      MPI_Comm           comm);
+                      const MPI_Comm &   comm);
```

Comment:
As mentioned in the other PR, I'm in favor of by-value passing. But I do like the `const` markup. That's correct.